### ISSUE TITLE
Remove "pipeline.output.workers" from doc and default logstash.yml

### ISSUE THREAD

**Issue by ppf2** [elastic/logstash#7884]
https://www.elastic.co/guide/en/logstash/5.5/logstash-settings-file.html

`pipeline.output.workers` doesn't do anything on 5.x but it's still documented as a configurable setting.

And it's also in the comments in the default logstash.yml:

```
# How many workers should be used per output plugin instance
#
# pipeline.output.workers: 1
#
```

**Comment by msspadoni** [comment]
I hope not to be wrong and bore you, but perhaps this is referenced also in the https://www.elastic.co/guide/en/logstash/current/performance-troubleshooting.html document at second point of section 4 ("Tune logstash worker settings").

**Comment by jordansissel** [comment]
@msspadoni Not a bore! Thank you for investigating :) This issue is a proposal to remove an unimplemented/unused setting `pipeline.output.workers`. 

What you refer to is `pipeline.workers` (no "output" part) and is a feature that will stay.

**Comment by dedemorton** [comment]
Closing because this is fixed in #8562 and #8908
